v8/plantuml_markdown/README.md

```diff
@@ -0,0 +1,109 @@
+This plugin renders [PlantUML](https://plantuml.com/) in Markdown files.
+
+# Requirements
+
+* [plantuml plugin](https://plugins.getnikola.com/#plantuml) (the actual PlantUML rendering is managed there)
```

Comment:
Create a `requirements-plugins.txt` file and list it there (as just `plantuml`, [example](https://github.com/getnikola/plugins/blob/master/v8/pkgindex/requirements-plugins.txt))